LabBuilder/LabBuilder.psm1

```diff
@@ -798,8 +798,8 @@ function Download-LabResources {
     # Downloading Lab Resources
     Write-Verbose -Message $($LocalizedData.DownloadingLabResourcesMessage)
 
-    # Bootstrap Nuget
-    $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
+    # Bootstrap Nuget # This needs to be a test, not a force 
+    # $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
```

Comment:
When executing on my system it paused for confirmation and then couldn't find PSRepository